x/committee/types/codec.go

```diff
@@ -92,7 +101,16 @@ func RegisterInterfaces(registry types.InterfaceRegistry) {
 		"kava.committee.v1beta1.PubProposal",
 		(*PubProposal)(nil),
 		&Proposal{},
+		&distrtypes.CommunityPoolSpendProposal{},
 		&govtypes.TextProposal{},
+		&upgradetypes.SoftwareUpgradeProposal{},
+		&proposaltypes.ParameterChangeProposal{},
+		&upgradetypes.SoftwareUpgradeProposal{},
+		&upgradetypes.CancelSoftwareUpgradeProposal{},
+	)
+
+	registry.RegisterImplementations(
+		(*govtypes.Content)(nil),
 		&CommitteeChangeProposal{},
```

Comment:
Moved these to register with the gov interface since they are handled through the gov msg type